script/seccomp.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# shellcheck source=./script/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")/lib.sh"
+
+# Due to libseccomp being LGPL we must include its sources,
+# so download, install and build against it.
+# Parameters:
+#  $1 -- libseccomp version to download and build.
+#  $2 -- destination directory to put the source tarball in.
+#  $3 -- file to append LIBSECCOMP_PREFIX*= environment variables to
+#        (can be sourced to get install paths).
+#  $@ -- additional architectures to cross-compile for.
+function build_libseccomp() {
+	local ver="$1"
+	shift
+	local dest="$1"
+	shift
+	local varfile="$1"
+	shift
+	local arches=("$@")
+	local tar="libseccomp-${ver}.tar.gz"
+
+	# Download.
+	wget "https://github.com/seccomp/libseccomp/releases/download/v${ver}/${tar}"{,.asc}
+
+	# Build and install natively.
+	tar xf "$tar"
+	pushd "libseccomp-${ver}" || return
```

Comment:
It would be nice if we didn't put this directory in the current directory and instead used a temporary directory, but this is already an existing issue with the current script so it doesn't hurt to fix it some other time.

Reply:
done